packages/visualisation/src/components/HoverInfoBox/index.js

```diff
@@ -82,6 +82,10 @@ const cargoName = (vehicleType) => {
 }
 
 const CarInfo = ({ data }) => {
+  const fill =
```

Comment:
Thinking about this, this may not be the way to go for all cars? F.x. if we have a long distance bus, they might take both cargo and passengers, I think we should have two progress bars instead of combining them, but if capacity is zero, not show that type.

Reply:
Sounds smart!